dspace-spring-rest/src/test/java/org/dspace/app/rest/builder/AbstractBuilder.java

```diff
@@ -60,8 +73,18 @@ public static void init() {
         resourcePolicyService = AuthorizeServiceFactory.getInstance().getResourcePolicyService();
         indexingService = DSpaceServicesFactory.getInstance().getServiceManager().getServiceByName(IndexingService.class.getName(),IndexingService.class);
         bitstreamFormatService = ContentServiceFactory.getInstance().getBitstreamFormatService();
+        registrationDataService = EPersonServiceFactory.getInstance().getRegistrationDataService();
+        versionHistoryService = VersionServiceFactory.getInstance().getVersionHistoryService();
+
+        // Temporarily disabled
```

Comment:
Quick note here
The "XMLWorkflow" specific services are currently commented out.
We will need to make sure that the spring config for the BasicWorkflow and XMLWorkflow are not clashing during the tests since they use the same service name in the getInstance method "workflowServiceFactory"